src/main/java/uk/gov/pay/api/model/Charge.java

```diff
@@ -86,6 +88,32 @@ public Charge(String chargeId, Long amount, PaymentState state, String returnUrl
         this.authorisationSummary = authorisationSummary;
     }
 
+    public Charge(String chargeId, Long amount, PaymentState state, String returnUrl, String description,
```

Comment:
Any reason for a new constructor here rather than just adding agreement id to the previous constructor? It's a boxed type so there should be no issue in it being constructed with a `null` value if it hasn't been passed in

Having a new constructor for every new expected field doesn't seem scalable